terminfo/models.txt

```diff
@@ -19,6 +20,7 @@ linux
 pcansi
 rxvt,rxvt-256color,rxvt-88color,rxvt-unicode,rxvt-unicode-256color
 screen,screen-256color
+screen.xterm,screen.xterm-88color,screen.xterm-256color|screen
```

Comment:
Fix for Screen (when run within xterm)

Reply:
models.txt is really not used anymore.  I used it to generate a subset of the database, but on its own it carries no value.  Also the "|screen" component looks fishy to me.